golem-worker-executor-base/src/preview2/mod.rs

```diff
@@ -50,16 +51,40 @@ impl From<golem_wasm_rpc::WitValue> for golem::rpc::types::WitValue {
     }
 }
 
+impl From<golem::rpc::types::WitValue> for golem_wasm_rpc::WitValue {
+    fn from(value: golem::rpc::types::WitValue) -> Self {
+        unsafe { mem::transmute(value) }
+    }
+}
+
 impl From<golem_wasm_rpc::Value> for golem::rpc::types::WitValue {
     fn from(value: golem_wasm_rpc::Value) -> Self {
         let wit_value: golem_wasm_rpc::WitValue = value.into();
         wit_value.into()
     }
 }
 
+impl From<golem::rpc::types::WitValue> for golem_wasm_rpc::Value {
+    fn from(value: golem::rpc::types::WitValue) -> Self {
+        let wit_value: golem_wasm_rpc::WitValue = value.into();
+        wit_value.into()
+    }
+}
+
 impl From<ValueAndType> for golem::rpc::types::WitValue {
     fn from(value: ValueAndType) -> Self {
         let wit_value: golem_wasm_rpc::WitValue = value.into();
         wit_value.into()
     }
 }
+
+impl From<self::wasi::clocks::wall_clock::Datetime> for DateTime<Utc> {
+    fn from(value: self::wasi::clocks::wall_clock::Datetime) -> DateTime<Utc> {
+        DateTime::from_timestamp(value.seconds as i64, value.nanoseconds)
+            .expect("Received invalid datetime from wasi")
+    }
+}
+
+// reexports so that we don't have to change version numbers everywhere
+pub use self::golem::api0_2_0 as golem_api_0_2_x;
```

Comment:
I was a bit annoyed with having to rename this everywhere when bumping the version, this should avoid it in the future

Reply:
👍